game-core/src/main/java/games/strategy/engine/chat/Chat.java

```diff
@@ -125,10 +125,10 @@ void updateStatus(final String status) {
   }
 
   String getStatus(final PlayerName playerName) {
-    return chatters.entrySet().stream()
-        .filter(n -> n.getKey().getPlayerName().equals(playerName))
+    return chatters.stream()
+        .filter(n -> n.getPlayerName().equals(playerName))
         .findAny()
```

Comment:
Even if you're not convinced by my suggestion above, I'd appreciate if this lookup would be extracted to a private helper method